docs/reference/search/async-search.asciidoc

```diff
@@ -0,0 +1,197 @@
+to retrieve the results at a later stage. Also, the currently available search
+results are returned as part of the <<search-api-response-body,`response`>> object.
+
+[source,console-result]
+--------------------------------------------------
+{
+  "id" : "FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=", <1>
+  "version" : 0,
+  "is_partial" : true, <2>
+  "is_running" : true, <3>
+  "start_time_in_millis" : 1583945890986,
+  "expiration_time_in_millis" : 1584377890986,
+  "response" : {
+    "took" : 1122,
+    "timed_out" : false,
+    "num_reduce_phases" : 0,
+    "_shards" : {
+      "total" : 562, <4>
+      "successful" : 3, <5>
+      "skipped" : 0,
+      "failed" : 0
+    },
+    "hits" : {
+      "total" : {
+        "value" : 157483, <6>
+        "relation" : "gte"
+      },
+      "max_score" : null,
+      "hits" : [ ]
+    }
+  }
+}
+--------------------------------------------------
+// TESTRESPONSE[s/FmRldE8zREVEUzA2ZVpUeGs2ejJFUFEaMkZ5QTVrSTZSaVN3WlNFVmtlWHJsdzoxMDc=/$body.id/]
+// TESTRESPONSE[s/1583945890986/$body.start_time_in_millis/]
+// TESTRESPONSE[s/1584377890986/$body.expiration_time_in_millis/]
+// TESTRESPONSE[s/"took" : 1122/"took": $body.response.took/]
+// TESTRESPONSE[s/"total" : 562/"total": $body.response._shards.total/]
+// TESTRESPONSE[s/"successful" : 3/"successful": $body.response._shards.successful/]
+// TESTRESPONSE[s/"value" : 157483/"value": $body.response.hits.total.value/]
+
+<1> Identifier of the async search that can be used to monitor its progress, retrieve its results, and/or delete it.
+<2> Whether the returned search results are partial or final
+<3> Whether the search is still being executed or it has completed
+<4> How many shards the search will be executed on, overall
+<5> How many shards have successfully completed the search
+<6> How many documents are currently matching the query, which belong to the shards that have already completed the search
+
+It is possible to block and wait until the search is completed up to a certain
+timeout by providing the `wait_for_completion` parameter, which defaults to
+`1` second.
+
+The Submit Async Search API supports the same <<search-search-api-query-params,parameters>>
+as the search API, though some have different default values:
+`pre_filter_shard_size` defaults to `1`, `batched_reduce_size` defaults to
+`5`, `request_cache` defaults to `true` and `ccs_minimize_roundtrips`
+defaults to `false`.
+
+It is also possible to specify how long the async search needs to be
+available through the `keep_alive` parameter, which defaults to `5d` (five days).
+Running async searches as well as saved search results are cleared once their
+corresponding `keep_alive` expires.
+
+WARNING: Async search does not support scroll searches, nor search requests that
+only include the `suggest` section. Cross cluster searches can be executed using
+async search but only with `ccs_minimize_roundtrips` disabled.
```

Comment:
```suggestion
WARNING: Async search does not support <<request-body-search-scroll,scroll>> or <<search-suggesters,suggesters>>. {ccs} is supported with <<ccs-min-roundtrips,`ccs_minimize_roundtrips`>> set to
`false`.
```

Reply:
it does support suggesters, but not requests that only ask for suggestions. confusing, right? Can you confirm @jimczi ?

Reply:
That's correct @javanna

Reply:
Ah. Thanks for clarifying. I'll resolve this. Please ignore the suggestion.